.changes/unreleased/tinymce-TINY-10781-2024-04-22.yaml

```diff
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Improved
+body: Editor now renders the currect high contrast color in Windows High Contrast
+  Mode
```

Comment:
```suggestion
body: Editor now renders correct high contrast colors in Windows High Contrast
  Mode
```

Reply:
Remember to add `.` at the end @ArvinJ-H 😄